### PR TITLE
Prefer set properties

### PR DESCRIPTION
**Pull request by balloob**
Prefer setProperties over setting individual properties on objects.

### CODE REVIEW

File: src/dialogs/more-info/controls/more-info-alarm_control_panel.html [hunk 1]
```diff
@@ -97,25 +93,27 @@
 
   stateObjChanged(newVal, oldVal) {
     if (newVal) {
-      this.codeFormat = newVal.attributes.code_format;
-      this.codeInputVisible = this.codeFormat !== null;
-      this.codeInputEnabled = (
+      const props = {
+        codeFormat: newVal.attributes.code_format,
+        armHomeButtonVisible: newVal.state === 'disarmed',
+        armAwayButtonVisible: newVal.state === 'disarmed',
```
Comment (c727) [inline src/dialogs/more-info/controls/more-info-alarm_control_panel.html]: we also have armCustomBypass

Comment (c727) [reply src/dialogs/more-info/controls/more-info-alarm_control_panel.html]: ... and night https://github.com/home-assistant/home-assistant-polymer/pull/760

Comment (balloob) [reply src/dialogs/more-info/controls/more-info-alarm_control_panel.html]: Out of scope for this PR. This is just translating it to use `setProperties`. I made no changes to the logic.

File: src/layouts/partial-cards.html [hunk 2]
```diff
@@ -210,27 +211,24 @@
...
-    constructor() {
-      super();
+    ready() {
       this.handleWindowChange = this.handleWindowChange.bind(this);
-      this.mqls = [300, 600, 900, 1200].map(function (width) {
-        var mql = window.matchMedia('(min-width: ' + width + 'px)');
-        mql.addListener(this.handleWindowChange);
-        return mql;
-      }.bind(this));
+      this.mqls = [300, 600, 900, 1200].map(width => matchMedia('(min-width: ' + width + 'px)'));
```
Comment (c727) [inline src/layouts/partial-cards.html]: \`(min-width: ${width}px))\`

File: src/dialogs/more-info/controls/more-info-alarm_control_panel.html [hunk 3]
```diff
@@ -97,25 +93,27 @@
...
+      };
+      props.codeInputEnabled = (
```
Comment (c727) [inline src/dialogs/more-info/controls/more-info-alarm_control_panel.html]: arlo also has the state `armed`

`props.codeInputEnabled = props.disarmButtonVisible || newVal.state === 'disarmed'`

File: src/layouts/partial-cards.html [hunk 4]
```diff
@@ -210,27 +211,24 @@
...
-        mql.removeListener(this.handleWindowChange);
-      });
+      this.mqls.forEach(mql => mql.removeListener(this.handleWindowChange));
     }
 
     handleWindowChange() {
-      var matchColumns = this.mqls.reduce(function (cols, mql) {
-        return cols + mql.matches;
-      }, 0);
+      var matchColumns = this.mqls.reduce((cols, mql) => cols + mql.matches, 0);
```
Comment (andrey-git) [inline src/layouts/partial-cards.html]: var -> const

**Review by andrey-git — APPROVED** [review]

**Pull request merged**